thaomonster/ACNH-Collections
# Feature/header

### What does this PR do? 
>Clearly and concisely describe the feature. 

This PR adds the Header to App and styles the Header component.

### Is this a feature or Fix/Refactor?  
Feature

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.

None.

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?

`src/Components/App/App.js` lines: 2,7;
`src/Components/Header/Header.js` lines: 1-21;
`src/Components/Header/_Header.scss ` lines: 1-27;

### How should this be tested?
>What steps need to be taken to confirm functionality? 
1. `git fetch` the branch
2. run `npm i `
3. run `npm start`
4. Look at the styling

### Steps To Take
>Any questions and possible steps moving forward.

Maybe put text under the image so it is known what it will link to.

### This PR is related to which issues:
>Tag issue here: `Closes #<issue_number>`
Closes #2
